data/transactions/logic/eval_test.go

```diff
@@ -2339,11 +2342,17 @@ func TestExtractFlop(t *testing.T) {
 	// fails in compiler
 	testProg(t, `byte 0xf000000000000000
 	extract
-	len`, 5, Expect{2, "extract expects 2 immediate arguments"})
+	len`, 5, Expect{2, "extract expects 3 stack arguments but stack height is 1"})
```

Comment:
Part of me wants this to say "extract expects 3 stack arguments or 2 immediates, but stack height is 1 and there are no immediates" (or similar but more succinct).

The only way I can imagine doing that is to try the other pseudo ops when the stack type checker fails, to see if they come "closer" to being correct, and somehow merging their complaints.  Seems too annoying for now.

As an alternative, I wonder if when a pseudo op complains in this way, we should identify it as `extract (no immediates)` rather than simple `extract`.